engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -66,13 +66,14 @@ namespace dmGameSystem
 
     static struct BlendModeParticleToGui
     {
-    	dmGui::BlendMode m_Table[4];
+    	dmGui::BlendMode m_Table[5];
     	BlendModeParticleToGui()
     	{
     		m_Table[dmParticleDDF::BLEND_MODE_ALPHA]		= dmGui::BLEND_MODE_ALPHA;
     		m_Table[dmParticleDDF::BLEND_MODE_MULT]			= dmGui::BLEND_MODE_MULT;
     		m_Table[dmParticleDDF::BLEND_MODE_ADD]			= dmGui::BLEND_MODE_ADD;
     		m_Table[dmParticleDDF::BLEND_MODE_ADD_ALPHA]	= dmGui::BLEND_MODE_ADD_ALPHA;
+            m_Table[dmParticleDDF::BLEND_MODE_SCREEN]       = dmGui::BLEND_MODE_SCREEN;
```

Comment:
The indentation looks weird here

Reply:
For future reference, is there a standard on the use of tabs and spaces in the source code. The new line I added uses spaces but the existing lines use tabs. It looks fine in my editor.

Reply:
Oh, I see. We should use spaces everywhere, but it may have degenerated in places over time as different people have worked on the code.

We should set up a code style check as one of the checks done for each commit and PR.

Sidenote: Personally I prefer tabs, but let's not go there :-)